Objects/memoryobject.c

```diff
@@ -1778,12 +1781,15 @@ pack_single(char *ptr, PyObject *item, const char *fmt)
     double d;
     void *p;
 
+    CHECK_RELEASED_INT(self); /* See gh-92888 for why we need this here */
```

Comment:
I don't think that it's worth it to do the check here. Doing it just before the final write is enough, no?